src/core/packages/saved-objects/import-export-server-internal/src/export/saved_objects_exporter.test.ts

```diff
@@ -1039,6 +1039,130 @@ describe('getSortedObjectsForExport()', () => {
         })
       );
     });
+
+    // Unskip after: https://github.com/elastic/kibana/issues/242671
+    describe.skip('access control', () => {
+      test('applies the access control transform if defined', async () => {
+        // const accessControlExportTransform: SavedObjectsExportTransform = (ctx, objects) => {
```

Comment:
We don't need to comment out the transform, the test is skipped anyway.
In fact, without the transform, this test doesn't actually test the implementation.